src/main/kotlin/it/pagopa/ecommerce/helpdesk/controllers/HelpdeskController.kt

```diff
@@ -27,6 +27,8 @@ class HelpdeskController(
     ): Mono<ResponseEntity<SearchTransactionResponseDto>> =
         helpDeskSearchTransactionRequestDto.flatMap {
             when (it) {
+                // TODO change logic here, search first into eCommerce then into PM DB when
+                // searching for criteria present in both DBs
```

Comment:
Is this TODO ok?

Reply:
Yes, I will add this logic in a separate PR. This one takes care only of search against PM database